keyboards/ergodox_ez/keymaps/skug/keymap.c

```diff
@@ -0,0 +1,278 @@
+/* Copyright 2018 Andreas Lindhé, Christoffer Holmberg
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+
+
+
+
+#include QMK_KEYBOARD_H
```

Comment:
For the includes, all you should need is: 
```c
#include QMK_KEYBOARD_H
#include "keymap_swedish.h"
```

All of the other files are included by default.

Reply:
`version.h` is still required

Reply:
Okay, I had a feeling that it might be.